src/services/doppler-api-client.ts

```diff
@@ -9,7 +9,8 @@ export interface DopplerApiClient {
   getSubscriber(email: string, apikey: string): Promise<ResultWithoutExpectedErrors<Subscriber>>;
   getSubscriberSentCampaigns(
     email: string,
-    apikey: string,
+    campaignsPerPage: number,
```

Comment:
How about adding some test for this mappings?

Reply:
i added